internal/http/services/owncloud/ocdav/trashbin.go

```diff
@@ -51,6 +52,59 @@ func (h *TrashbinHandler) init(c *Config) error {
 	return nil
 }
 
+func (h *TrashbinHandler) handleTrashbinSpaces(s *svc, w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	log := appctx.GetLogger(ctx)
+
+	var spaceID string
+	spaceID, r.URL.Path = router.ShiftPath(r.URL.Path)
+
+	_, base, ok := spaces.DecodeSpaceID(spaceID)
+	if !ok {
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
+	log.Debug().Str("path", base).Msg("decoded space base path")
+
+	u := appctx.ContextMustGetUser(ctx)
+
+	if r.Method == MethodPropfind {
+		h.listTrashbin(w, r, s, u, base, "", "")
+		return
+	}
+
+	var key string
+	key, r.URL.Path = router.ShiftPath(r.URL.Path)
+	if key != "" && r.Method == MethodMove {
+		// find path in url relative to trash base
```

Comment:
Is this commented code useful any longer?

Reply:
I guess this is now handled by `extractDestination`, so it can be removed